bftengine/src/bftengine/BftCommandHandler.h

```diff
@@ -0,0 +1,26 @@
+#include <Replica.hpp>
+
+#pragma once
+
+namespace bftEngine {
+
+class BftCommandHandler : public IRequestsHandler {
+ private:
```

Comment:
Conventions: (sorry for being nagging) :
please put public at a first place and members should end with an '_' underscore